CMakeLists.txt

```
@@ -16,24 +16,85 @@ project(
     LANGUAGES CXX
 )
 
-#---- C++ standard that we need (at least).
-set(CMAKE_CXX_STANDARD          20   CACHE BOOL "" FORCE)
-set(CMAKE_CXX_STANDARD_REQUIRED True CACHE BOOL "" FORCE)
+#---- Options.
+option(KOKKOSUTILS_ENABLE_TESTS "Enable testing" ON)
+option(KOKKOSUTILS_ENABLE_DOC   "Enable documentation" ON)
 
-#---- Find Kokkos.
-find_package(
-    Kokkos
-    ${KOKKOS_VERSION}
-    REQUIRED
+#---- Global property that helps us keep track of files that will automatically be added
+#     to our Doxygen documentation.
+define_property(GLOBAL PROPERTY FILES_FOR_DOC 
```

```suggestion
define_property(GLOBAL PROPERTY KOKKOSUTILS_FILES_FOR_DOC 
```
(to avoid conflicts with someone using your library)